syntax = "proto3";

// First message from MPF to platform
message GetPlatformDetailsRequest {
  string mpf_version = 1;
}

// Platform returns available hardware and initial switch states
message GetPlatformDetailsResponse {
  map<string, bool> known_switches_with_initial_state = 1;
  repeated string known_lights = 2;
  repeated string known_coils = 3;
}
